.github/workflows/tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: "3.8"
+        python-version: "3.9"
```

Comment:
https://github.com/executablebooks/rst-to-myst/actions/runs/11331640903/job/31512111103